### DISCUSSION TITLE
GDPR: Add support for Google Maps "nocookie" version (maps.googleapis.com)

### DISCUSSION THREAD

**Discussion by mircobabini** [elementor/elementor#21415] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

When activating GDPR Iubenda with the legal option "Block scripts to prevent... etc" the Google Maps are not showing up.

### Describe the solution you'd like

Easy as using a different domain from Google Maps which is maps.googleapis.com instead of www.google.com/maps or maps.google.com. There's a workaround:

https://gist.github.com/mircobabini/167a11c8cdb4e925ec69e6c759301aca

**Comment by m1ga** [comment]
you are not allow to show any google maps without user consent. You could use e.g. Real Cookie Banner ([tutorial](https://medium.com/p/19ecffe4339c)) or my simple ["click to map" plugin](https://wordpress.org/plugins/click-to-map-for-elementor/)

**Reply by mircobabini** [reply]
Why not? Until I use the "nocookie" mode is basically the same as using GA4 with anonymized IP which is legal for statistics purposes even if the user consent is not given. Not installing cookies nor profiling the user. Isn't it legal?

**Reply by m1ga** [reply]
anonymized IPs are only anonymized on their server so you already made a connection to it. You aren't even allowed to use Google Fonts (no cookie) without permission. I'm not a lawyer but that's how I do my client pages: no external calls without permission, no GA, TagManager, Maps,... everything that already calls an URL makes you give away their IP without permission.
